mkdocs.yml

```diff
@@ -0,0 +1,15 @@
+site_name: ddev Documentation
```

Comment:
Does this file have to go in the root? Lots of apps are now moving (like circleci and travis have) to putting things in dedicated subdirs). If that's an option, it would be nicer.